tests/languages/sparql/keyword_feature.test

```diff
@@ -0,0 +1,271 @@
+A
+ADD
+ALL
+AS
+ASC
+ASK
+BNODE
+BY
+CLEAR
+CONSTRUCT
+COPY
+CREATE
+DATA
+DEFAULT
+DELETE
+DESC
+DESCRIBE
+DISTINCT
+DROP
+EXISTS
+FILTER
+FROM
+GROUP
+HAVING
+INSERT
+INTO
+LIMIT
+LOAD
+MINUS
+MOVE
+NAMED
+NOT
+NOW
+OFFSET
+OPTIONAL
+ORDER
+RAND
+REDUCED
+SELECT
+SEPARATOR
+SERVICE
+SILENT
+STRUUID
+UNION
+USING
+UUID
+VALUES
+WHERE
+ABS(
+AVG(
+BIND(
+BOUND(
+CEIL(
+COALESCE(
+CONCAT(
+CONTAINS(
+COUNT(
+DATATYPE(
+DAY(
+ENCODE_FOR_URI(
+FLOOR(
+GROUP_CONCAT(
+HOURS(
+IF(
+IRI(
+isBLANK(
+isIRI(
+isLITERAL(
+isNUMERIC(
+isURI(
+LANG(
+LANGMATCHES(
+LCASE(
+MAX(
+MD5(
+MIN(
+MINUTES(
+MONTH(
+ROUND(
+REGEX(
+REPLACE(
+sameTerm(
+SAMPLE(
+SECONDS(
+SHA1(
+SHA256(
+SHA384(
+SHA512(
+STR(
+STRAFTER(
+STRBEFORE(
+STRDT(
+STRENDS(
+STRLANG(
+STRLEN(
+STRSTARTS(
+SUBSTR(
+SUM(
+TIMEZONE(
+TZ(
+UCASE(
+URI(
+YEAR(
+
+----------------------------------------------------
+
+[
+		["keyword", "A"],
+		["keyword", "ADD"],

```

Comment:
I forgot to say this last time: Your editor seems to have incorrectly detected the indentation as 2 spaces before converting to tabs. Please fix the indentation in, I think, all test files.

Reply:
No problem. Thanks, updated the testfiles that had wrong indentation.